source/Mission.cpp

```diff
@@ -679,6 +679,13 @@ bool Mission::HasFailed(const PlayerInfo &player) const
 
 
 
+bool Mission::HasFailed() const
```

Comment:
We should name this and the above method differently, since this one is just a getter, while the above actually does things. `IsFailed`?